src/components/menu/menu-item/menu-item.style.ts

```diff
@@ -85,9 +88,17 @@ const StyledMenuItemWrapper = styled.a.attrs({
       `}
     }
 
-    a button:not(.search-button) {
-      position: relative;
-      top: -1px;
+    ${hasIcon &&
+    !maxWidth &&
+    css`
+      ${StyledContent} {
+        position: relative;
+        top: -2px;
+      }
+    `}
```

Comment:
**suggestion**: what do you think about using the `:has()` selector for this styling scenario? This would eliminate the need for the `hasIcon` React prop.

```tsx
css`
  &:has([data-component="icon"]) ${StyledContent} {
    ${!maxWidth &&
    css`
      position: relative;
      top: -2px;
    `}
  }
`
```